ibm/data_source_ibm_cis_healthchecks.go

```diff
@@ -115,6 +115,26 @@ func dataSourceIBMCISHealthChecks() *schema.Resource {
 							Type:     schema.TypeInt,
 							Computed: true,
 						},
+						cisGLBHealthCheckHeader: {
+							Type:     schema.TypeSet,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									cisGLBHealthCheckHeaderHeader: {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									cisGLBHealthCheckHeaderValues: {
+										Type:     schema.TypeSet,
+										Computed: true,
+										Elem: &schema.Schema{
+											Type: schema.TypeString,
+										},
+									},
+								},
+							},
+							Set: hashByMapKey(cisGLBHealthCheckHeaderHeader),
```

Comment:
not required if type list is used

Reply:
it did not work after changing to TypeList

Reply:
modified